src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs

```diff
@@ -88,22 +88,24 @@ class OnefuzzNamingPolicy : JsonNamingPolicy {
     }
 }
 public class EntityConverter {
```

Comment:
we might as well make this a `static` class

Reply:
yeah that was my initial intent but keep it a class allow us to inject a request specific logger.